releasenotes/notes/remove-duplicated-python-code-d4e1e29f420206cd.yaml

```diff
@@ -0,0 +1,6 @@
+---
+deprecations:
+  - |
+    The core Agent check Python code is no longer duplicated here and is instead
+    pulled from integrations-core. The code now resides in the `datadog_checks`
+    namespace, though the old `checks`, `utils`, etc. paths are still supported.
```

Comment:
can you add something like: `Please update your custom checks accordingly` , and add a link to the relevant documentation?